Formula/gdal.rb

```diff
@@ -119,7 +97,7 @@ def install
       "--with-cfitsio=#{Formula["cfitsio"].opt_prefix}",
       "--with-hdf5=#{Formula["hdf5"].opt_prefix}",
       "--with-netcdf=#{Formula["netcdf"].opt_prefix}",
-      "--with-jasper=#{Formula["jasper"].opt_prefix}",
+      "--with-openjpeg",
```

Comment:
We don't need to specify the location here?

Reply:
No, the `--with-openjpeg` flag does not accept any argument